### ISSUE TITLE
Dynamic themes

### ISSUE THREAD

**Issue by rafarovina** [react-toolbox/react-toolbox#1305]
Hello,

I'm migrating from Material UI to React-Toolbox and I can't find a way of having dynamic themes with it.
I needed a simple way of overriding the default primary/accent color.
I tried using react-css-themr and some other tools. I was only able to change the theme via webpack, but then, it's not dynamic.

Any suggestions on this?
Thanks.

**Comment by redaxmedia** [comment]
Does this help? https://github.com/react-toolbox/react-toolbox#theming-configuration-variables

**Comment by rafarovina** [comment]
@redaxmedia , not really. I needed something to set the theme on the run.
Like [Material UI](http://www.material-ui.com/#/customization/themes) does with ligh/dark theme.

**Comment by henryruhs** [comment]
You can load PostCSS with browserify to process it on runtime but sounds like a bad idea :D

**Comment by seivan** [comment]
@rafarovina If you're using `react-css-themr` shouldn't you just be able to switch out `theme` in 
```
<ThemeProvider theme={theme}>
```
?

**Comment by rafarovina** [comment]
@seivan Yes, I'm able to change theme of specif components but I'm not able to change the primary color (for all components) using `react-css-themr`.
The only approach I found was to manipulate theme of all components with my own sass/JS variables.

**Comment by czebe** [comment]
So what is the proper way to switch themes on the fly / during runtime? Docs are not very clear on this.
